pkg/status/console_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright Contributors to the Open Cluster Management project
+package status
+
+import (
+	"testing"
+
+	bpv1 "github.com/stolostron/backplane-operator/api/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_mapConsoleDeployment(t *testing.T) {
+	tests := []struct {
+		name string
+		ds   *appsv1.Deployment
+		want bpv1.ComponentCondition
+	}{
+		{
+			name: "healthy deployment",
+			ds: &appsv1.Deployment{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "operator.open-cluster-management.io/v1",
+					Kind:       "Deployment",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-console",
+				},
+				Status: appsv1.DeploymentStatus{
+					Conditions: []appsv1.DeploymentCondition{
+						{
+							Type:    appsv1.DeploymentAvailable,
+							Status:  corev1.ConditionTrue,
+							Reason:  "Available",
+							Message: "deployment available",
+						},
+					},
+				},
+			},
+			want: bpv1.ComponentCondition{
```

Comment:
I don't fully understand this status condition. It seems like it reports "OCP Console missing", but only if the console deployment is present and available?

Reply:
I agree this was incorrect. The ConsoleUnavailableStatus will only be called in `ensureNoConsoleMCE` if ocp's console is also disable, so I've changed to have the same status outcome and message after setting the `ToggledOffStatus` in `ensureNoConsoleMCE` as well. I think this is correct since it emmulates the same logic we're using in MCH now too